app/assets/stylesheets/blacklight/_constraints.scss

```diff
@@ -2,7 +2,7 @@
   @extend .mb-2;
 }
 
-span.constraints-label {
+.constraints-label {
```

Comment:
Is this only used to style this `.sr-only` element?

https://github.com/projectblacklight/blacklight/blob/e7d7cb883e4d24a5263797cac47a8e5fd5b603ca/app/views/catalog/_constraints.html.erb#L4